_i18n/en.yml

```diff
@@ -801,7 +801,24 @@ roadmap:
   p2pool: P2Pool released
   haveno: Haveno
 
-
+  ringsize-sera-tryp: increase ring size to 100+ through Seraphis or Triptych
```

Comment:
I'm not sure if this is the best way to show this. If a milestone is unclear, better not be specific and instead use something generic like ` increase ring size to >100`